docs/common/craft-parts/reference/part_properties.rst

```diff
@@ -90,6 +90,22 @@ to the ``bin`` directory in the staging area and renamed to ``hello``:
    organize:
      hello.py: bin/hello
 
+If partitions are in use by the application, they may be referenced by prepending the partition name, surrounded by parentheses, to the destination path.  Only the destination path may specify a partition; source paths always reference the ``default`` partition.  Omitting the partition name in the destination path causes the file to be copied to the ``default`` partition.
```

Comment:
Stuff that mentions partitions should have a way to ignore them when including the file (e.g. a header so we can have an `end_before`).

Reply:
@lengau I don't understand what you mean.  I'm going to mark this resolved for now, I expect to make another pass on the docs as I implement partitions and we can address this then.

Reply:
The stuff that's in the `common` directory gets `include`d in other projects. Rockcraft for example includes these files but doesn't have partitions enabled, so we should have a way to exclude the info about partitions from the Rockcraft docs.

Reply:
Ahh ok gotcha, thank you!  I'll do this on the next round.